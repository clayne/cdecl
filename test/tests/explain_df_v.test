cdecl @ /dev/null @ -xc++ @ explain [[deprecated]] void f() @ 0
