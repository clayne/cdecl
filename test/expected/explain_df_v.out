declare f as deprecated function returning void
